apps/docs/pages/guides/database/tables.mdx

```diff
@@ -425,7 +425,7 @@ Views provide the several benefits:
 
 #### Simplicity
 
-As a query becomes complex it becomes a hassle to call it. Especially when we run it at regularly. In the example above, instead of repeatedly running:
+As a query becomes complex it becomes a hassle to call it. Especially when we run it regularly. In the example above, instead of repeatedly running:
```

Comment:
```suggestion
As a query becomes more complex, it can be a hassle to call it over and over - especially when we run it regularly. In the example above, instead of repeatedly running:
```